stdlib/public/Platform/Platform.swift

```diff
@@ -103,8 +103,7 @@ public var errno : Int32 {
   get {
 #if os(OSX) || os(iOS) || os(watchOS) || os(tvOS) || os(FreeBSD)
     return __error().pointee
-//FIXME: os(Windows) should be replaced, such as triple(Cygwin)
```

Comment:
I'm worried that "env" is too vague.  At its broadest, basically everything I can imagine testing for with #if is a difference in the "programming environment".  If we say it's narrower than that, then what does it mean, exactly, beyond "something that's historically been written in the environment field of target triples"?  Because I'm not sure that that's really all that consistent across different targets, and I'm very hesitant about just re-exposing the rather weird design of triples.